tutorials/notebooks/embedding_tokens.ipynb

```diff
@@ -83,24 +79,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|██████████| 2/2 [00:00<00:00, 5419.00it/s]\n",
-      "100%|██████████| 2/2 [00:00<00:00, 6786.90it/s]"
+      "100%|██████████| 2/2 [00:00<00:00, 6600.01it/s]\n",
+      "100%|██████████| 2/2 [00:00<00:00, 9521.69it/s]"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "This is the token vocabulary we created: \n",
+      "This is the token_ids vocabulary we created: \n",
       "\n",
-      "{0: '@@PADDING@@', 1: '@@UNKNOWN@@', 2: 'All', 3: 'the', 4: 'cool', 5: 'kids', 6: 'use', 7: 'character', 8: 'embeddings', 9: '.', 10: 'I', 11: 'prefer', 12: 'word2vec', 13: 'though', 14: '...'}\n",
+      "{0: '@@PADDING@@', 1: '@@UNKNOWN@@'}\n",
```

Comment:
This looks like it's wrong - why did all of the vocab items go missing?  Did you forget to change how you queried the vocab?  Or maybe there was some old state hanging around somewhere?